clients/constants.go

```diff
@@ -58,3 +58,10 @@ const (
 	SystemManagementAgentServiceKey = "edgex-sys-mgmt-agent"
 	SupportSchedulerServiceKey      = "edgex-support-scheduler"
 )
+
+const (
+	ContentType       = "Content-Type"
+	ContentTypeBinary = "application/cbor"
```

Comment:
Since the other two constants end with the actual encoding mechanism (e.g. Json, and Yaml), shouldn't this one too?

Also in general, names which include initialisms (or acronyms) should uses consistent case (e.g. "URL" or "url", not "urL" or "Url"). See [Go Code Review comments](https://github.com/golang/go/wiki/CodeReviewComments#initialisms).